keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -320,95 +351,49 @@ needed.
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
-
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+API servers with feature enabled will experience different latency and throughput of LIST requests.
```

Comment:
Make it explicit that it's not a workload feature (so can't impact already running workloads).

Also, make it expicit that now LIST requests can be failing (if watchcache is lagging).

Reply:
Done